action/protocol/staking/candidate_center.go

```diff
@@ -52,16 +87,23 @@ func listToCandChange(l CandidateList) (*candChange, error) {
 }
 
 // NewCandidateCenter creates an instance of CandidateCenter
-func NewCandidateCenter(all CandidateList) (*CandidateCenter, error) {
+func NewCandidateCenter(all CandidateList, opts ...CandidateCenterOption) (*CandidateCenter, error) {
```

Comment:
for candidate center, which may be constructed from https://github.com/iotexproject/iotex-core/blob/master/action/protocol/staking/candidate_statemanager.go#L84, how could the current code return a CandidateCenter with the right parameter?

Reply:
this will be done in PR3688